spot-client/src/common/app-state/spot-tv/reducer.js

```diff
@@ -20,6 +20,7 @@ const DEFAULT_STATE = {
     tileView: false,
     videoMuted: true,
     view: undefined,
+    volumeControlSupported: undefined,
```

Comment:
false?

Reply:
Well, I thought undefined means it wasn't defined explicitly, but yeah, false makes sense